index.js

```diff
@@ -0,0 +1,301 @@
+'use strict';
+
+/*   /\ ,__,   ,__, /\
+    / / |  |   |  | \ \
+   / /  |  |   |  |  \ \
+  / /   |  '---'  |   \ \
+  \ \   |  ,---,  |   / /
+   \ \  |  |   |  |  / /  Composable and joyful
+    \ \ |__|   |__| / /  HTML document traversal
+     \/             */
+
+//. # sanctuary-html
+//.
+//. TK.
```

Comment:
Is documentation the only remaining task?

Reply:
I believe so. :)

Reply:
Oh, can this be released as beta, so I can try even without docs?

Reply:
@geekyi, you should be able to depend on the unpublished code by adding the following dependency to your project's __package.json__:

```json
{
  "dependencies": {
    "sanctuary-html": "sanctuary-js/sanctuary-html#d934957391eef4df59fd0b7ba2556c2c2adec97b"
  }
}
```

Let me know whether this is an acceptable solution. :)